sdk/handleError.js

```diff
@@ -0,0 +1,40 @@
+const get = require('lodash.get');
+const { inspect } = require('util');
+const rp = require('request-promise-native');
+
+const logger = require('../lib/logger');
+
+const { translateErrorCode } = require('binaris-pickle');
+
+class APIError extends Error {}
+
+async function callAPI(options, type = 'post') {
```

Comment:
`callAPI` is a really bad name. It's both generic and does not explain what it does.
How about something like loggedRequest of verboseRP or requestWithNiceErrors, or ...
(you get it)

Reply:
Consider it done.